lib/logout.js

```diff
@@ -209,9 +206,9 @@ module.exports.logout = function (options) {
 
       // Logout flow in progress, incoming SAMLResponse from SP. (Could be SP initiated or IdP initiated)
       } else if (req.query.SAMLResponse || req.body.SAMLResponse) {
-        function processLogoutResponse(logoutResponse, transactionId) {
+        function processLogoutResponse(state, transactionId) {
```

Comment:
Then can you change the name of the function? It's doing nothing with logout response then